rho/clicommands.py

```diff
@@ -563,12 +497,29 @@ def _do_command(self):
                   "Please use --reset with profile first.")
             sys.exit(1)
 
-        cmd_string = 'ansible-playbook rho_playbook.yml -i data/'\
-                     + profile + '_hosts ' + '-v -f ' + forks
+        if facts == ['default']:
+            facts_to_collect = 'default'
+        elif os.path.isfile(facts[0]):
+            facts_to_collect = _read_in_file(facts[0])
+        else:
+            assert isinstance(facts, list)
+            facts_to_collect = facts
+
+        ansible_vars = {'facts_to_collect': facts_to_collect,
+                        'report_path': report_path}
+
+        cmd_string = ('ansible-playbook rho_playbook.yml '
+                      '-i data/{profile}_hosts -v -f {forks} '
+                      '--extra-vars \'{vars}\'').format(
+                          profile=profile,
+                          forks=forks,
+                          vars=json.dumps(ansible_vars))
 
         # process finally runs ansible on the
         # playbook and inventories thus created.
 
+        print('Running:', cmd_string)
```

Comment:
`print('Running:', cmd_string)` <-- Seems like debug information that we should log instead of print to stdout.